test/integration/cli/exit-codes.test.js

```diff
@@ -127,6 +127,7 @@ describe('CLI exit codes', () => {
 
 	describe('when the `level` config is set to "warning" but the `--level` flag is set to "notice"', () => {
 
+		/* Should add many more rules to ignore..
```

Comment:
Is there a reason this test case it commented out? I think we should add the rules to ignore so we have a working test

Reply:
Too long list in my opinion and maintaining it later...